web/html/src/components/pagination.js

```diff
@@ -43,6 +47,37 @@ const ItemsPerPageSelector = (props) =>
   </select>
 ;
 
+const PageSelector = (props) => {
+  var range = [];
+  for (var i = 1; i <= props.lastPage; i++) {
+    range.push(i);
```

Comment:
Yuck, maybe you can take inspiration from https://stackoverflow.com/questions/3895478/does-javascript-have-a-method-like-range-to-generate-a-range-within-the-supp and make this range variable go away completely

Reply:
Good improvement. There are several *nice* way of doing it, I am a fun of `Array.from...` so I used this. We could also use something from `lodash` but for this simple case it is not the case to import a module for one line only. Fixed now. :+1: